azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -88,7 +104,9 @@ def test_device_connect(self):
         command = "iot central device compute-device-key --pk {} -d {}".format(
```

Comment:
Would be **great** to eliminate the need for azext_iot_central_primarykey and azext_iot_central_scope_id simplifying test setup. This `test_device_connect` could create a new device and grab the key for it and scope ID from the API. Thoughts?

In general, we need to keep an eye out for other test setup simplifications.

Reply:
we have open work items for the next sprint. they'll cover a lot of code refactoring and simplifications